contracts/StakeManager.sol

```diff
@@ -54,16 +54,19 @@ contract StakeManager is IStakeManager {
         emit StakeWithdrawn(relayManager, msg.sender, amount);
     }
 
-    function authorizeHub(address relayManager, address relayHub) external override {
-        StakeInfo storage info = stakes[relayManager];
-        require(info.owner == msg.sender, "not owner");
+    modifier ownerOrManagerOnly(address relayManager) {
+        StakeInfo storage infoByManager = stakes[relayManager];
+        StakeInfo storage infoBySender = stakes[msg.sender];
+        require(infoByManager.owner == msg.sender || infoBySender.owner != address(0), "not owner or manager");
```

Comment:
Note the bug discussed here (for the record)

Reply:
Fixed